qualitycontrol.sh

```diff
@@ -31,8 +34,10 @@ cmake $SOURCEDIR                                              \
       ${MONITORING_VERSION:+-DMonitoring_ROOT=$MONITORING_ROOT} \
       ${CONFIGURATION_VERSION:+-DConfiguration_ROOT=$CONFIGURATION_ROOT} \
       ${INFOLOGGER_VERSION:+-DInfoLogger_ROOT=$INFOLOGGER_ROOT} \
+      ${O2_VERSION:+-DO2_ROOT=$O2_ROOT}                       \
```

Comment:
Why making it optional here and not below?

Reply:
Actually most of them should be optional below as well (I guess) or make them non optional here.

Reply:
yes, agreed. @knopers8 could you fix that?

Reply:
@ktf Sure. We agreed with @Barthelemy to get rid of all options here. Just give me a while to make sure it doesn't break anything

Reply:
@ktf done